dotnet/src/Microsoft.AutoGen/Agents/AgentBase.cs

```diff
@@ -171,6 +171,8 @@ public List<string> Subscribe(string topic)
                 }
             }
         };
+
+        // TODO: Should this be async?
```

Comment:
was purposefully not sync - but should have a try/catch/error handling or retry - we want to make sure that subscriptions complete because if we think they are there but aren't there yet we may miss messages we thought we were getting...probably better ways to handle this.